4_advanced_usage/47_node_properties_data_sources.txt

```diff
@@ -0,0 +1,71 @@
+=== Node properties data source
+
+As explain in previous chapters, Nodes have properties that can be
+used to create groups or in techniques and directives parameters.
+These properties are key/value pairs, with value being a simple
+string or a well formed JSON document. The simplest way to define a
+property for a Node is thanks Rudder REST API.
+
+Rudder 4.1 introduces a new way to define Node properties by defining
+data sources.
+
+The following schema explains the general working process of data source:
+
+image:./images/rudder-datasources-description.png[Data source description]
```

Comment:
This diagram is great!!!

May I suggest a few changes:
- In the top right orange box, remove "CMDB" in the title (so it's "API Response") (CMDB is an example)
- In the sentence "Retrieve node properties from company CMDB" please replace "company" with "organisation"
- In the example config in that same box, the authentication token looks like it would be different for each node. This is a very unlikely example. Please replace it with ${rudder.parameters.cmdb-token}
- Please replace "other nodes properties" with "other node properties" (using the plural in English makes it seem like properties from DIFFERENT nodes would be included here!)
- The square red box that says "API" should also have a "UI" box next to it, since we can configure these via the UI. This could put people off using it.
- Minor detail: in the list of available trigger, "Interactive" should not have a capital "i"